buildpipeline/pipeline.json

```diff
@@ -163,6 +202,20 @@
         {
           "Name": "Core-Setup-CrossBuild",
           "Parameters": {
+            "PB_DockerTag": "ubuntu1404_cross_prereqs_v2",
+            "PB_Architecture": "arm",
+            "TargetRid": "linux-arm",
+            "PB_PortableBuild": "-portable"
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "Ubuntu 14.04",
```

Comment:
Just like RHEL7.2 is base minimum for linux-x64, Ubuntu 14.04 was determined the same by @janvorli.

Also, i am not aware of RHEL for arm.